index.html

```diff
@@ -35,10 +35,78 @@ <h2>
       Online resources
     </h2>
     <ul>
-      <li><a href="https://freecodecamp.org">Free Code Camp</a> - free online coding courses and community</li>
       <li><a href="https://glitch.com">Glitch</a> - online code editor which allows you to ask for help with your code online</li>
       <li><a href="https://www.codenewbie.org/">Code Newbie</a> - community and podcast for people learning to code</li>
       <li><a href="http://coder-newbie.surge.sh/">Even longer (open source) list of free resources</a></li>
+      <li><a href="https://www.hackerrank.com/">HackerRank</a></li>
```

Comment:
Might be nice to add explanation e.g. "Free coding exercises in lots of languages and developer job interview question practice"